tcms/xmlrpc/tests/test_backward_compatibility.py

```diff
@@ -1,100 +0,0 @@
-# -*- coding: utf-8 -*-
-
-from django_nose.testcases import FastFixtureTestCase
-
-from tcms.management.models import Product
-from tcms.management.models import TestBuild
-from tcms.testcases.models import TestCase
-from tcms.testplans.models import TestPlan
-from tcms.testruns.models import TestCaseRun
-from tcms.testruns.models import TestRun
-from tcms.xmlrpc.serializer import ProductXMLRPCSerializer
-from tcms.xmlrpc.serializer import TestBuildXMLRPCSerializer
-from tcms.xmlrpc.serializer import TestCaseRunXMLRPCSerializer
-from tcms.xmlrpc.serializer import TestCaseXMLRPCSerializer
-from tcms.xmlrpc.serializer import TestPlanXMLRPCSerializer
-from tcms.xmlrpc.serializer import TestRunXMLRPCSerializer
-from tcms.xmlrpc.serializer import XMLRPCSerializer
-
-
-__all__ = ('TestSerializationBackwardCompatibility',)
-
-
-serializers_classes = {
-    Product: ProductXMLRPCSerializer,
-    TestBuild: TestBuildXMLRPCSerializer,
-    TestCase: TestCaseXMLRPCSerializer,
-    TestCaseRun: TestCaseRunXMLRPCSerializer,
-    TestPlan: TestPlanXMLRPCSerializer,
-    TestRun: TestRunXMLRPCSerializer,
-}
-
-
-def remove_alias_extra_fields(model_class, serialize_result):
-    '''Have to remove extra fields, these are unnecessary for this test'''
-
-    serializer_class = serializers_classes[model_class]
-
-    if not hasattr(serializer_class, 'extra_fields'):
-        return
-
-    if 'alias' not in serializer_class.extra_fields:
-        return
-
-    alias_fields = serializer_class.extra_fields['alias']
-    for original_name, mapped_name in alias_fields.iteritems():
-        del serialize_result[mapped_name]
-
-
-class TestSerializationBackwardCompatibility(FastFixtureTestCase):
-    '''Ensure new serialization method to generate same data with original
-
-    Remove this test when migrate to new serialization completely.
```

Comment:
This test case was written for ensuring new serialization classes in ``xmlrpc/serializer.py`` work as expected and the API returns same serialized data as before (before the change). This test case should had been removed long time ago, but it's still there as of this PR. So, remove it.